cfme/rest/__init__.py

```diff
@@ -354,3 +355,23 @@ def mark_vm_as_template(rest_api, provider, vm_name):
         lambda: rest_api.collections.templates.find_by(name=vm_name).subcount != 0,
         num_sec=700, delay=15)
     return rest_api.collections.templates.get(name=vm_name)
+
+
+def arbitration_settings(request, rest_api, num=2):
```

Comment:
@mfalesni @psav So should I rename it to something like "collections"? Should I do it as separate PR once this one is merged?